src/ripple/app/consensus/RCLValidations.h

```diff
@@ -0,0 +1,207 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_APP_CONSENSUSS_VALIDATIONS_H_INCLUDED
+#define RIPPLE_APP_CONSENSUSS_VALIDATIONS_H_INCLUDED
+
+#include <ripple/basics/ScopedLock.h>
+#include <ripple/consensus/Validations.h>
+#include <ripple/protocol/Protocol.h>
+#include <ripple/protocol/STValidation.h>
+#include <vector>
+
+namespace ripple {
+
+class Application;
+
+/** Wrapper over STValidation for generic Validation code
+
+    Wraps an STValidation::pointer for compatibility with the generic validation
+    code.
+*/
+class RCLValidation
+{
+    STValidation::pointer val_;
+public:
+
+    /** Constructor
+
+        @param v The validation to wrap.
+    */
+    RCLValidation(STValidation::pointer const& v) : val_{v}
+    {
+    }
+
+    /// Validated ledger's hash
+    uint256
+    ledgerID() const
+    {
+        return val_->getLedgerHash();
+    }
+
+    /// Validated ledger's sequence number (0 if none)
```

Comment:
I see I wasn't consistent with that for `loadFee`.  I'll add the explicit check.